app/feature/feature-addon-purchase/src/main/kotlin/com/hedvig/android/feature/addon/purchase/ui/summary/AddonSummaryDestination.kt

```diff
@@ -202,30 +213,97 @@ private fun SummarySuccessScreen(uiState: Content, onConfirmClick: () -> Unit, n
 @Composable
 private fun SummaryCard(uiState: Content, modifier: Modifier = Modifier) {
   val locale = getLocale()
-  val formattedDate = remember(uiState.summaryParameters.activationDate, locale) {
+  val formattedDate = remember(uiState.activationDate, locale) {
     HedvigDateTimeFormatterDefaults.dateMonthAndYear(
       locale,
-    ).format(uiState.summaryParameters.activationDate.toJavaLocalDate())
+    ).format(uiState.activationDate.toJavaLocalDate())
   }
   QuoteCard(
     subtitle = stringResource(R.string.ADDON_FLOW_SUMMARY_ACTIVE_FROM, formattedDate),
-    premium = uiState.summaryParameters.quote.price.toString(),
-    displayItems = uiState.summaryParameters.quote.addonVariant.displayDetails.map {
-      QuoteDisplayItem(
-        it.first,
-        null,
-        it.second,
-      )
+    premium = @Composable {
```

Comment:
If we remove the annotation here, does it work ok?
```suggestion
    premium = {
```